demos/demo_onejoint_print_position.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+"""Send zero-torque commands to Finger robot and print joint positions."""
+import numpy as np
+
+from robot_interfaces import one_joint
+import blmc_robots
+
+home_offset = 2.241742
+robot_data = one_joint.Data()
+robot_backend = blmc_robots.create_one_joint_backend("can6",
+                                                     home_offset,
+                                                     robot_data)
+robot = one_joint.Frontend(robot_data)
+
+
+robot_backend.initialize()
+
+# TODO this makes the application hang
+#t = finger.get_current_time_index()
+#print("t = %d" % t)
```

Comment:
Sorry, I missed this the first time, the commented stuff here could also be removed